src/nvim/lua/treesitter.c

```diff
@@ -13,6 +13,7 @@
 #include <stdlib.h>
 #include <string.h>
 #include <tree_sitter/api.h>
+#include <tree_sitter/parser.h>
```

Comment:
This is not part of the public tree-sitter API (it's meant for parsers, not consumers), afaik. We would have to hack the cmake file to pull in that header as well.

Reply:
Makes sense, maybe not worth it for such a tiny change